docs/tutorial/nangang.md

```diff
@@ -0,0 +1,36 @@
+# Nangang Test
+
+Nangang is the first internal testnet. See
+https://github.com/vacp2p/research/issues/43 for more.
+
+## How to
+
+Build:
+
+```
+make wakunode2
```

Comment:
This can be removed as `scripts2` target depends on `wakunode2` target.

Reply:
Indeed, I left it there deliberately because the second target might change and not build wakunode.